.github/workflows/issue.yml

```diff
@@ -0,0 +1,92 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: ISSUE Standardized Process
+
+on:
+  schedule:
+    - cron: '0 0 */1 * *'  # once a day
+
+  issue_comment:
+    types: [created,edited]
+
+  issues:
+    types: [labeled,edited,reopened]
+  
+
+jobs:
+  comment-action:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'issue_comment' }}
+    steps:
+      - name: Print Issue Comment
+        run: |
+          echo ${{ github.event.comment.body }}
+
+      - name: Trigger The CI Workflow
+        if: ${{ contains( github.event.comment.body , '/run ci') }} 
+        uses: mvasigh/dispatch-action@main
+        with:
+          token: ${{ secrets.PERSONAL_TOKEN }}
+          repo: shardingsphere
+          owner: ${{ github.actor }}
+          event_type: rerun-ci
+
+  remove-inactive:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'issue_comment' || ( github.event_name == 'issues' && ( github.event.action == 'edited' || github.event.action == 'reopened' ) ) }}
+    steps:
+    - name: Remove "status:inactive" label
+      if: ${{ contains( github.event.issue.labels.*.name , format('status{0} inactive', ':') ) }}
+      uses: actions-cool/issues-helper@v2.2.1
+      with:
+        actions: 'remove-labels'
+        token: ${{ secrets.GITHUB_TOKEN }}
+        issue-number: ${{ github.event.issue.number }}
+        labels: "status: inactive"
+
+  
+  check-inactive-issue:
+    runs-on: ubuntu-latest
+    concurrency: check-inactive  # singleton-run-stage
+    if: ${{ github.event_name == 'schedule' }}
```

Comment:
Also need to judge whether it is opened?

Reply:
By default, only these open issues will be checked.